google/cloud/storage/internal/async/reader_connection_tracing.cc

```diff
@@ -52,22 +52,24 @@ class AsyncReaderConnectionTracing
         .then([count = ++count_, span = span_](auto f) -> ReadResponse {
           auto r = f.get();
           if (absl::holds_alternative<Status>(r)) {
-            span->AddEvent("gl-cpp.read",
-                           {
-                               {sc::kMessageType, "RECEIVED"},
-                               {sc::kMessageId, count},
-                               {sc::kThreadId, internal::CurrentThreadId()},
-                           });
+            span->AddEvent(
+                "gl-cpp.read",
+                {
+                    {/*sc::kRpcMessageType=*/"rpc.message.type", "RECEIVED"},
+                    {/*sc::kRpcMessageId=*/"rpc.message.id", count},
```

Comment:
Ditto, if these names are no longer part of the semantic conventions, maybe we should drop them?  Or maybe there is a newer name to represent these things?

Reply:
These names are the new hotness. The symbols are not available in the oldest OTel we support (`v1.9.1`), so we hardcode their values.